src/components/Grid.vue

```diff
@@ -40,7 +40,7 @@ export default {
 
     // Each Column Definition results in one Column.
     const columnDefs = reactive({
-      value: [{ field: "id" }, { field: "title" }, { field: "text" }],
+      value: props.gridColumn,
```

Comment:
I would set columnDefs `onMounted` like we are with rowData. That way we know that props are ensured to be updated